federation/backends/minimal_profile.md

```diff
@@ -0,0 +1,194 @@
+# Minimal profile for openEO backends
+This document describes the requirements for a backend that aims to provide minimal functionality for access and processing of gridded
+earth observation data.
+
+This document does not aim to contradict the openEO API, and when this is perceived to be the case, the definitions in the openEO API get precedence.
+It is possible that requirements in this document do define additional constraints on top of the openEO API.
+
+This profile is established because the openEO web service API is quite liberal if interpreted strictly. So a web service can be compliant to the API 
+specification, but still be mostly incompatible with other backends, and existing usage patterns. By following this profile, implementors have a better
+view on what is important.
+
+## Scope
+This minimal profile aims to help initial openEO implementations, that seek a least effort approach to achieve minimal compatibility with other backends.
+Requirements in this profile are not sufficient for backends in the openEO platform federation.
+
+## Mandatory endpoints
+
+In the openEO API, all endpoints are optional and discoverable. These are the endpoints that are required by this profile:
+
+
+
+```
+[
+    {
+      "methods": [
+        "GET"
+      ],
+      "path": "/file_formats"
+    },
+    {
+      "methods": [
+        "GET"
+      ],
+      "path": "/collections"
+    },
+    {
+      "methods": [
+        "GET"
+      ],
+      "path": "/collections/{collection_id}"
+    },
+    {
+      "methods": [
+        "GET"
+      ],
+      "path": "/processes"
+    },
+    {
+      "methods": [
+        "GET",
+        "POST"
+      ],
+      "path": "/jobs"
+    },
+    {
+      "methods": [
+        "GET",
+        "DELETE"
+      ],
+      "path": "/jobs/{job_id}"
+    },
+    {
+      "methods": [
+        "GET",
+        "POST",
+        "DELETE"
+      ],
+      "path": "/jobs/{job_id}/results"
+    },
+    {
+      "methods": [
+        "GET"
+      ],
+      "path": "/jobs/{job_id}/results/{user_base64}/{secure_key}"
+    },
+    {
+      "methods": [
+        "GET"
+      ],
+      "path": "/jobs/{job_id}/results/items/{user_base64}/{secure_key}/{item_id}"
+    },
+    {
+      "methods": [
+        "GET"
+      ],
+      "path": "/jobs/{job_id}/results/items/{item_id}"
+    },
+    {
+      "methods": [
+        "GET"
+      ],
+      "path": "/jobs/{job_id}/results/assets/{filename}"
+    },
+    {
+      "methods": [
+        "GET"
+      ],
+      "path": "/jobs/{job_id}/results/assets/{user_base64}/{secure_key}/{filename}"
+    }
+  ],
+```
+
+## Batch job support
+This backend mandates support for batch jobs over the synchronous jobs via the /result endpoint.
+Both are very relevant, but batch jobs are not constrained in processing time.
+
+## File formats
+GeoTiff is the only mandatory output format: https://docs.openeo.cloud/federation/backends/fileformats.html#geotiff
```

Comment:
This should be a relative link in markdown format so that VuePress can pick it up.